pkgs/by-name/ch/chatzone-desktop/package.nix

```
@@ -0,0 +1,70 @@
+{ lib
+, appimageTools
+, runCommand
+, fetchurl
+, stdenvNoCC
+, makeDesktopItem
+, copyDesktopItems
+, makeWrapper
+}:
+
+let
+  pname = "chatzone-desktop";
+  version = "5.1.10";
+
+  src = fetchurl {
+    url = "https://cdn1.ozone.ru/s3/chatzone-clients/ci/20240129/254/chatzone-desktop-linux-5.1.10.AppImage";
+    hash = "sha256-8jmJyg+Y2n8jGpWYnBRFsnCPIzoggJxG7neF7g6QQwk=";
+  };
+
+  appimage = appimageTools.wrapType2 {
+    inherit pname version src;
+  };
+  appimageContents = appimageTools.extract {
+    inherit pname version src;
+  };
+  icon = runCommand "chatzone-app-icon.png" {} ''
+    cp ${appimageContents}/app_icon.png $out
+  '';
+in stdenvNoCC.mkDerivation {
+  inherit pname version;
+  src = appimage;
+
+  nativeBuildInputs = [ copyDesktopItems makeWrapper ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      inherit icon;
+      name = "chatzone";
+      desktopName = "Chatzone";
+      comment = "Ozon corporate messenger";
+      exec = pname;
```

```suggestion
      exec = "chatzone-desktop";
```
Using pname like this is an anti pattern

---

Thanks for the insight, fixed it